src/Components/Articles.jsx

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useState } from "react";
+import Article from "./Article";
+
+function Articles() {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    fetch("https://chris-reddit.onrender.com/api/articles")
+      .then((res) => {
+        return res.json();
```

Comment:
the articles can be set directly in here, we dont need to pass it to the next then block